ng2-components/ng2-alfresco-upload/src/components/file-uploading-list.component.spec.ts

```diff
@@ -210,18 +210,45 @@ describe('FileUploadingListComponent', () => {
     });
 
     describe('uploadErrorFiles()', () => {
-        it('returns the error files', () => {
-            file.status = 6;
+        it('returns array of error files', () => {
```

Comment:
Please check all tests are compatible with the latest convention: https://github.com/Alfresco/alfresco-ng2-components/blob/development/CONTRIBUTING.md#test-guide